### ISSUE TITLE
[WEB] Error: The string supplied did not seem to be a phone number

### ISSUE THREAD

**Issue by aytunch** [natintosh/plugin_libphonenumber#23]
Hi @natintosh 
I use the below api with `number = '4'` and `isoCode='en'`
I know that `4` is not a valid number but the user is typing in to a search box and I have to call parse at every key stroke.
`final parsedNumber = await PhoneNumberUtil.normalizePhoneNumber(number, isoCode);`
and this throws an exception. Is this expected?

<img width="780" alt="Screenshot 2023-03-02 at 00 39 00" src="https://user-images.githubusercontent.com/6442915/222270812-3670a0f1-4432-4600-b3b9-8dd39f2feb98.png">

**Comment by natintosh** [comment]
Yes... Just wrap it in try catch. When it's unable to parse a number it's throws an exception

**Comment by aytunch** [comment]
@natintosh thanks for the answer. In `libphonenumber` I guess they handled this in the plugin code. When I migrated to your plugin these exceptions appeared. I still want to suggest we have the same behavior for the sake of consistency. What do you think? best regards.
